main.py

```diff
@@ -260,6 +260,282 @@ async def on_message_edit(before, after):
+
+            # Looks up the dictionary and adds the
+            # correspponding morse code
+            # along with a space to separate
+            # morse codes for different characters
+            cipher += MORSE_CODE_DICT[letter] + " "
+        else:
+            # 1 space indicates different characters
+            # and 2 indicates different words
+            cipher += " "
+
+    return cipher
+
+
+# Function to decrypt the string
+# from morse to english
+def decrypt(message):
+
+    # extra space added at the end to access the
+    # last morse code
+    message += " "
+
+    decipher = ""
+    citext = ""
+    for letter in message:
+
+        # checks for space
+        if letter != " ":
+
+            # counter to keep track of space
+            i = 0
+
+            # storing morse code of a single character
+            citext += letter
+
+        # in case of space
+        else:
+            # if i = 1 that indicates a new character
+            i += 1
+
+            # if i = 2 that indicates a new word
+            if i == 2:
+
+                # adding space to separate words
+                decipher += " "
+            else:
+
+                # accessing the keys using their values (reverse of encryption)
+                decipher += list(MORSE_CODE_DICT.keys())[list(
+                    MORSE_CODE_DICT.values()).index(citext)]
+                citext = ""
+
+    return decipher
+
+
+@bot.message_command(name="Encrypt to Morse")
+async def _tomorse(ctx, message: discord.message):
+    result = encrypt(message.content.upper())
+    await ctx.send(result)
+
+
+@bot.message_command(name="Decrypt Morse")
+async def _frommorse(ctx, message: discord.message):
+    result = decrypt(message.content)
+    await ctx.send(result)
+
+
+@bot.message_command(name="Decrypt binary")
+async def _frombinary(ctx, message: discord.message):
+
+	if message.content.lower() == "01000000 01100101 01110110 01100101 01110010 01111001 01101111 01101110 01100101":
+		await ctx.respond("SMH. Allowed mentions are turned off. go do something better.")
+	else:
+		a_binary_string = message.content
+		binary_values = a_binary_string.split()
+
+		ascii_string = ""
+		for binary_value in binary_values:
+			an_integer = int(binary_value, 2)
+
+			ascii_character = chr(an_integer)
+
+			ascii_string += ascii_character
+
+		await ctx.send(ascii_string,
+					allowed_mentions=discord.AllowedMentions.none())
+
+
+@bot.message_command(name="Encrypt to binary")
+async def _tobinary(ctx, message: discord.message):
+	if message.content.lower() == 'bruce':
+		await ctx.respond("01010000 01101111 01100111 01100111 01100101 01110010 01110011 00101110")
+	elif message.content.lower() == 'easter egg':
+		await ctx.respond("01010000 01110010 01101111 01100010 01100001 01100010 01101100 01111001 00100000 01101110 01101111 01110100")
+
+	elif message.content.lower() == '@everyone':
+		await ctx.respond("Wow, you though allowed mentions were on? Smh.")
```

Comment:
This is childish, and doesn't even work as intended as someone could easily add another character to bypass this. Same goes for all of the other checks here.

Reply:
They aren't checks, just little easter eggs

Reply:
An if statement is a check.
![Screen Shot 2021-10-01 at 10 27 58 AM](https://user-images.githubusercontent.com/71356958/135646703-1a0eda95-5219-4fe9-9868-55fa16c4468b.png)